include/picongpu/particles/shapes/P4S.hpp

```diff
@@ -101,17 +101,17 @@ struct P4S : public shared_P4S::P4S
              */
             float_X const abs_x = algorithms::math::abs( x );
 
-            bool const below_2nd_radius = abs_x < 1.5_X;
-            bool const below_1st_radius = abs_x < 0.5_X;
+            bool const isWithin_2nd_radius = abs_x < 1.5_X;
+            bool const isWithin_1st_radius = abs_x < 0.5_X;
```

Comment:
I think this is okay, would have personally prefered camelCase-conformant `isWithinSecondRadius`. I understand other variable names are also not all in camelCase.